com.unity.render-pipelines.universal/Runtime/2D/Renderer2D.cs

```diff
@@ -49,6 +49,20 @@ public Renderer2D(Renderer2DData data) : base(data)
 
             m_UseDepthStencilBuffer = data.useDepthStencilBuffer;
 
+            // Hook in the debug-render where appropriate...
+            m_Render2DLightingPass.DebugHandler = DebugHandler;
+            m_FinalBlitPass.DebugHandler = DebugHandler;
+
+            if (m_PostProcessPasses.finalPostProcessPass != null)
+            {
+                m_PostProcessPasses.finalPostProcessPass.DebugHandler = DebugHandler;
+            }
+
+            if (m_PostProcessPasses.postProcessPass != null)
+            {
+                m_PostProcessPasses.postProcessPass.DebugHandler = DebugHandler;
```

Comment:
As this targets latest trunk, just suggestion =>m_PostProcessPasses.finalPostProcessPass?.DebugHandler = DebugHandler; (I like C# new features :D, it is up to u ofcourse)

Reply:
Yeah I like them too! Though in this case I'll leave it like this for two reasons: 1) this code is already removed in the follow-up PR so I don't feel like refactoring it, and 2) I think your example actually wouldn't work here because it's an assignment. I think you could do it with a setter like this: `m_PostProcessPasses.postProcessPass?.SetDebugHandler(DebugHandler)`. Anyway point is moot here :)